src/config.yaml

```diff
@@ -7,6 +7,14 @@ ADAFRUIT_IO:
   ADAFRUIT_IO_KEY: 'ENTER YOUR ADAFRUIT_IO KEY HERE'
   FEEDNAME: 'ENTER YOUR ADAFRUIT_IO FEEDNAME THAT YOU WANT TO MONITOR HERE'
 
+#Custom scripts. Add your scripts commands.
+Script:
```

Comment:
The actions in the config.yaml are in alphabetical order. This part should be moved down.